conf/rosalind.config

```diff
@@ -0,0 +1,28 @@
+params {
+	config_profile_description = 'Kings College London Rosalind HPC'
+	config_profile_contact = 'Theo Portlock'
+	config_profile_url = 'https://www.rosalind.kcl.ac.uk/'
+}
+
+singularity {
+	enabled = true
+	autoMounts = true
+	docker.enabled = false
+}
+
+params {
+	max_memory = 64.GB
```

Comment:
According to the docs, max memory would be `384.GB`, no?

https://rosalind.kcl.ac.uk/hpc/compute_nodes/

Reply:
I was under the impression that the 10 percent of total resource usage was their policy (https://rosalind.kcl.ac.uk/hpc/scheduler_policy/). I've just tested a job requiring 384.GB and am recieving the "Memory specification can not be satisfied" sbatch error. I have contacted a member of the Rosalind support team for further clarification and will update you when they reply. Thanks for the suggestion!

Reply:
Ah sorry, I thought that was across the entire cluster - I must have misunderstood. What you put makes sense then :+1: